{
	"101": {
		"0404": {
			"name": "PYTHON_CORE_INVALID_REFERENCE",
			"message": "",
			"short_description": "Unknown cause.",
			"long_description": "Unknown cause.",
			"short_solution": "Unknown solution.",
			"long_solution": "Unknown solution.",
			"support_link": "https://preten.do/101-0404"
		}
	}
}